File: specification/specs/common/PipelineFailure.ts
enum PipelineFailure {
	BadAuthentication = 0,
	BadResponse = 1,
	PingFailure = 2,
	SniffFailure = 3,
	CouldNotStartSniffOnStartup = 4,
	MaxTimeoutReached = 5,
	MaxRetriesReached = 6,
	Unexpected = 7,
	BadRequest = 8,
	NoNodesAttempted = 9
}

File: output/csharp/Common/PipelineFailure.cs

using System.Runtime.Serialization;

namespace Nest.Common {

	public enum PipelineFailure {
  
		[DataMember(Name = "BadAuthentication")] BadAuthentication,
		[DataMember(Name = "BadResponse")] BadResponse,
		[DataMember(Name = "PingFailure")] PingFailure,
		[DataMember(Name = "SniffFailure")] SniffFailure,
		[DataMember(Name = "CouldNotStartSniffOnStartup")] CouldNotStartSniffOnStartup,
		[DataMember(Name = "MaxTimeoutReached")] MaxTimeoutReached,
		[DataMember(Name = "MaxRetriesReached")] MaxRetriesReached,
		[DataMember(Name = "Unexpected")] Unexpected,
		[DataMember(Name = "BadRequest")] BadRequest,
		[DataMember(Name = "NoNodesAttempted")] NoNodesAttempted
	}
}
